about.html

```diff
@@ -0,0 +1,130 @@
+      </div>
+      <nav class="navbar container-main" id="navbar-top">
+        <a href="index.html" class="brand-name dtp clr-1">
+          <img src="images/logo-header.png" alt="Virtual Defense">
+        </a>
+        <button type="button" class="menu mob" id="mobile-menu"></button>
+        <nav class="navs dtp" id="nav-top-right">
+          <a href="index.html" class="navlink" id="nav-link-tr-1">Home</a>
+          <a href="#" class="navlink" id="nav-link-tr-2">Program</a> 
+          <a href="#" class="navlink" id="nav-link-tr-3">Join</a> 
+          <a href="#" class="navlink" id="nav-link-tr-4">Sponsor</a> 
+          <a href="#" class="navlink" id="nav-link-tr-5">News</a>
+          <a href="#" class="navlink" id="nav-link-tr-6">VD Campaign</a>        
+        </nav>
+      </nav>
+    </header>
+    <main>
+      <section id="headlineAbout">
+        <div class="container-main cnt-abt">
+          <p class='title-tag ff-2 clr-2 al-center'>"Hello! Securing world"</p>
+          <h1 class="about-head ff-1 clr-2 al-center" id="About-title">
+            VIRTUAL DEFENSE<br>GLOBAL SUMMIT 2022
+          </h1>
+          <div class="abt-desc" id="abt-main-desc">
+            <p class="sec-text ff-2 al-center" id="desc-text">
+              The need for data protection, Cyber Security and beyond. 
+              Here from global cyber security experts on the best practices
+              to achieve optimum.
+            </p>
+            <div class="pattern-2" id="desc-pat-tr-2"></div>
+          </div>
+          <p class="al-center" id="duration">
+            For More Details and participation, please contact us at <br>
+            vd-summit-contacs&commat;gmail.com
+          </p>
+        </div>             
+      </section>
+      <section class="abt-sec">
+        <div class="container-main abt-logo cnt-abt">
+          <h2 class="head4 ff-2 al-center">
+            VIRTUAL DEFENSE<br>GLOBAL SUMMIT 2022 LOGO
+          </h2>
+          <hr>
+          <img src="images/logoMain.png" alt="VD Logo" id="brand-logo">
+        </div>
+      </section>
+      <section class="abt-sec">
+        <div class="container-main abt-logo cnt-abt" id = 'past-summit'>
+          <h2 class="head4 ff-2 al-center">
+            See the past VD Global Summits
+          </h2>
+          <hr>
+          <div class="summit-cnt">
+            <div class="summits" id="summit-1">
+              <h2 class="clr-4">
+                2019
+              </h2>
+              <p class="clr-4">
+                VD Global Summit 2019 in London
+              </p>
+            </div>
+            <div class="summits" id="summit-2">
+              <h2 class="clr-4">
+                2020
+              </h2>
+              <p class="clr-4">
+                VD Global Summit 2020 in Munich
+              </p>
+            </div>
+          </div>
+          
+        </div>
+      </section>
+    </main>
+    <footer class="footer bg-dark" id="abt-footer">
```

Comment:
Great use of semantics 😎 The proper use of tags is key to having better HTML! Well done using section and footer tags 💪